trillian/ctfe/configpb/config.proto

```diff
@@ -124,6 +124,9 @@ message LogConfig {
   // be signed by this log's private key, and will be verified using the public
   // key specified in this config.
   SignedTreeHead frozen_sth = 16;
+
+  // A list of X.509 extension OIDs which should cause submissions to be rejected.
```

Comment:
You could describe in this doc comment how the extension OID is formatted. Is it the dotted form e.g. "1.2.3.4"?

Reply:
Done